.github/workflows/image.yml

```diff
@@ -260,18 +200,9 @@ jobs:
     timeout-minutes: 30
 
     steps:
-      - name: Download Docker Image
-        uses: actions/download-artifact@v5
-        with:
-          pattern: symbolicator-image@amd64
-          path: /tmp
-
-      - name: Load Docker Image
-        run: docker load --input /tmp/symbolicator-amd64.tar
-
       - name: Run Sentry self-hosted e2e CI
         uses: getsentry/self-hosted@master
         with:
           project_name: symbolicator
-          image_url: symbolicator-amd64
+          image_url: ghcr.io/getsentry/symbolicator:${{ github.sha }}
```

Comment:
hmm not sure why this is resolvig to `ghcr.io/getsentry/symbolicator:fbf420b5b599548d6598d552965a99f42760783b`, doesnt even exist